servlet/src/test/scala/org/http4s/servlet/BlockingHttp4sServletSpec.scala

```diff
@@ -14,7 +14,7 @@ import scala.concurrent.duration._
 
 class BlockingHttp4sServletSpec extends Http4sSpec {
 
-  lazy val service = HttpRoutes.of[IO] {
+  lazy val service = HttpApp[IO] {
```

Comment:
This match is now non-exhaustive! Please add `.orNotFound` below.

Reply:
This was fixed in my latest commit